content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,235 @@
+Azure remote where I have pushed some datasets.
+
+I am now trying to read those datasets from another repository
+("my-project-repo"), using `dvc get` (e.g.
+`dvc get git@github.com:data-registry-repo.git path/data.csv`) but I get this
+error:
+
+```bash
+ERROR: failed to get 'path/data.csv' from 'git@github.com:data-registry-repo.git' - Authentication to Azure Blob Storage via default credentials (https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.4.0/azure.identity.html#azure.identity.DefaultAzureCredential) failed.
+Learn more about configuration settings at <https://man.dvc.org/remote/modify>: unable to connect to account for Must provide either a connection_string or account_name with credentials!!
+```
+
+---
+
+Generally, there are two ways solve this issue:
+
+- [ENV vars](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.environmentcredential?view=azure-python)
+- Setup some options using the `--global` or `--system` flags to update the DVC
+  config
+
+If you're going to update the DVC config to include your cloud credentials, use
+the `dvc remote modify` command. Here's an example of how you can do that with
+Azure using the `--global` flag.
+
+```dvc
+$ dvc remote modify --global myremote connection_string 'mysecret'
+```
+
+This will modify the global config file, _~/.config/dvc/config_, instead of the
+_.dvc/config_ file. You could also use the `--system` flag to modify the
+_/etc/xdg/dvc/config_ file if that's necessary for your project.
+
+### [Is there any way to ensure that `dvc import` uses the cache from the config file and how can I keep the cache consistent for multiple team members?](https://discord.com/channels/485586884165107732/563406153334128681/827574712825413672)
+
+This is another great question where a little context might be useful.
+
+---
+
+I'm trying to import a dataset project into another DVC project, let's call it
+_dvcdata_.
+
+The config for _dvcdata_ is:
+
+```ini
+[core]
+    remote = awsremote
+[cache]
+    type = symlink
+    dir = /home/user/dvc_cache
+['remote "awsremote"']
+    url = s3://...
+```
+
+When I run `dvc import git@github.com:user/dvcdata.git my_data`, it starts to
+redownload it. I have double checked that I have pushed this config file to
+master.
+
+---
+
+The repo you are importing into has its own cache directory. If you want to use
+the same cache directory across both projects, you have to configure _cache.dir_
+and _cache.type_ in both projects.
+
+You can set up the cache dir and cache link type in your own global config and
+then when project 1 imports _dvcdata_, it will be cached there. Finally when
+project 2 imports _dvcdata_, it will just be linked or copied, depending on the
+config, from the cache without downloading.
+
+We recommend you use the `--global` or `--system` flags in the `dvc config`
+command for all of your projects across all machines. An example of this would
```

Comment:
`for all of your projects` - this config option is global, it's not per project

`across all machines` - sounds like too much, we can skip it I would say